pipelines/common/utilities/Configure-1ES.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
```

Comment:
Eventually, I want to create an agent pool with images pre-provisioned with this script, so it can be removed from here.